dis_snek/models/discord/guild.py

```diff
@@ -1000,16 +998,14 @@ def get_channel(
             Channel object if found, otherwise None
 
         """
-        if channel_id in self._channel_ids or channel_id in self._thread_ids:
+        if channel_id in self._channel_ids:
```

Comment:
I think we should call the `to_snowflake` method to sanitise `channel_id`, bcu ppl may pass in as str instead of int, which may cause comparison issues.

Reply:
Sure.
Btw there are many times we missed this